docs/notebooks/util/plotting_plotly.py

```diff
@@ -153,6 +154,52 @@ def plot_gp_plotly(model, mins: TensorType, maxs: TensorType, grid_density=20):
     return fig
 
 
+def plot_dgp_plotly(
+    model, mins: TensorType, maxs: TensorType, grid_density: int = 20, num_samples: int = 100
+):
+    """
```

Comment:
We should add some explanation what this function does, I know we don't produce API for these functions and weren't documenting them properly (bad practice) but they will nevertheless be used by anyone who takes a look at tutorials